polaris.shopify.com/src/components/Markdown/components/DoDont/index.tsx

```diff
@@ -64,13 +68,27 @@ export const DoDont = ({children}: PropsWithChildren) => {
     <Fragment>
       {prefix}
       <div className={styles.DoDont}>
-        {dos.length ? (
-          <Box className={[styles.DoDontPart, styles.Do]}>{dos}</Box>
-        ) : null}
-        {donts.length ? (
-          <Box className={[styles.DoDontPart, styles.Dont]}>{donts}</Box>
-        ) : null}
+        {dos.length ? <Do title={null}>{dos}</Do> : null}
+        {donts.length ? <Dont title={null}>{donts}</Dont> : null}
       </div>
     </Fragment>
   );
 };
+
+export const Do = ({children, title = 'Do'}: DoDontProps) => {
+  return (
+    <Box className={[styles.DoDontPart, styles.Do]}>
+      {title && <h4>{title}</h4>}
```

Comment:
Because this is outside of the Markdown parser, it wont get an automatic id & "copy link" button applied. Is that ok?

Reply:
Checking against prod, there's no copy button there either.